src/NUnitFramework/framework/Internal/Execution/WorkItem.cs

```diff
@@ -90,6 +90,9 @@ public WorkItem(WorkItem wrappedItem)
             Result = wrappedItem.Result;
             Context = wrappedItem.Context;
             ParallelScope = wrappedItem.ParallelScope;
+#if PARALLEL
+            TestWorker = wrappedItem.TestWorker;
+#endif
```

Comment:
An earlier version of this was failing due to null TestWorker property. It's no longer relevant to the change, but I thought it best to leave this in.